tests/tag.in

```diff
@@ -0,0 +1,3 @@
+gitpython
+packaging
+typer
```

Comment:
Should we remove `hacking/tagger/requirements.txt`?

Reply:
I think that depends on if we want to support people running `tag.py` without the `nox` session, or if the intention is to deprecate that and have the only supported tagging method be `nox -e tag`.

Reply:
I'm fine with removing hacking/tagger/requirements.txt.

Reply:
Staging this.